src/Tools/BuildActionTelemetryTable/BuildActionTelemetryTable.csproj

```diff
@@ -31,6 +31,10 @@
 
   <ItemGroup>
     <Compile Include="..\..\Compilers\Core\Portable\InternalUtilities\Hash.cs" Link="Utilities\Hash.cs" />
+    <Compile Include="..\..\Dependencies\PooledObjects\ArrayBuilder.cs" Link="Utilities\ArrayBuilder.cs" />
+    <Compile Include="..\..\Dependencies\PooledObjects\ArrayBuilder.Enumerator.cs" Link="Utilities\ArrayBuilder.Enumerator.cs" />
+    <Compile Include="..\..\Dependencies\PooledObjects\ObjectPool`1.cs" Link="Utilities\PooledObjects\ObjectPool`1.cs" />
+    <Compile Include="..\..\Dependencies\PooledObjects\PooledHashSet.cs" Link="Utilities\PooledObjects\PooledHashSet.cs" />
```

Comment:
Did the new members/behavior on ArrayBuilder require adding these files?

Reply:
yes.  thsi was due to adding:

`internal static int Hash.CombineValues<T>(ArrayBuilder<T> values)`

Reply:
Note: all other consumers of Hash.cs already consumed the reset of these.  so this was just an annoying one-off tool we had that was brought in line with those.